includes/Admin/AdminNotices.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace VPlugins\BlogPostConnector\Admin;
+
+class AdminNotices {
+    public function __construct() {
+        add_action('admin_notices', [$this, 'permalinkNotice']);
+        add_action('network_admin_notices', [$this, 'multisiteWarning']);
+        add_action('admin_post_fix_permalink_structure', [$this, 'fixPermalinkStructure']);
+    }
+
+    public function multisiteWarning() {
+        if (is_multisite() && is_network_admin() && current_user_can('manage_network_plugins')) {
+            $screen = get_current_screen();
+            if ($screen && $screen->id === 'plugins-network') {
+                echo '<div class="notice notice-warning is-dismissible">';
+                echo '<p><strong>SM Blog Post Connector plugin should be activated individually on each subsite for full functionality.</strong></p>';
+                echo '</div>';
+            }
+        }
+    }
+
+    public function permalinkNotice() {
+        if (!current_user_can('manage_options')) return;
+
+        $permalink_structure = get_option('permalink_structure');
+        if ($permalink_structure !== '/%postname%/') {
+            $fix_url = wp_nonce_url(admin_url('admin-post.php?action=fix_permalink_structure'), 'fix_permalink');
+            echo '<div class="notice notice-error">';
+            echo '<p><strong>Warning:</strong> The permalink structure is not set to "Post name". This may cause routing issues for the Blog Post Connector plugin.</p>';
```

Comment:
Change this warning note like this 

Notice: The current permalink structure is set to “Plain”, which is not compatible with this plugin’s API routing. To ensure proper functionality, it’s recommended to switch to the “Post name” structure.
You can update it by clicking [here].
Caution: Changing the permalink structure may impact existing URLs on your site. Please proceed carefully.